src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -14,6 +18,8 @@ public Order completeOrder(Long userId) {
         // NOTE: In production ready code this order identifier should be generated by DB
         // For test purpose we simplify this and return dummy data
         order.setOrderId(1L);
+        logger.info("completeOrder method was called for User with ID: ({}) "
```

Comment:
The log message should be more informative. Include the details of the order that was completed. For example, you could log the total number of products or the total price of the order.